app/models/metrics/person_escort_records/counts_by_move_time_bin.rb

```diff
@@ -0,0 +1,33 @@
+module Metrics
+  module PersonEscortRecords
+    class CountsByMoveTimeBin
+      include BaseMetric
+      include PersonEscortRecords
+      include TimeBins
+
+      def initialize(supplier: nil)
+        setup_metric(
+          supplier: supplier,
+          label: 'PER counts by move time bin',
+          file: 'counts_by_move_time_bin',
+          interval: 5.minutes,
+          columns: {
+            name: COUNT,
+            field: :itself,
+            values: [COUNT],
```

Comment:
Just checking my understanding here - in the other metrics calculators we append to an array whereas here we replace the array. Is this because this is a timebound query which doesn't need to hold historical data? I assume the appending is to preserve historical data.

Reply:
Hi Nev, its because this particular metric only has one column (Count), e.g.:

<img width="369" alt="Screenshot 2021-04-27 at 14 06 23" src="https://user-images.githubusercontent.com/86833/116246332-cd1f9f00-a761-11eb-9b19-e5d6d5c94b15.png">


...whereas other metrics can have multiple columns, e.g.

<img width="1058" alt="Screenshot 2021-04-27 at 14 07 18" src="https://user-images.githubusercontent.com/86833/116246450-e9bbd700-a761-11eb-893f-5934cd48fe36.png">